src/GameLoop_test.py

```diff
@@ -1,5 +1,48 @@
-from src.GameLoop import takeTurns
+import sys
+sys.path
+sys.path.append("/src/")
 
+from GameLoop import takeTurns
+from TicTacToe import GameState
+from unittest import mock
+import pytest
 
-def test_dummy():
-    return False
\ No newline at end of file
+
+def test_Greeting(capsys):
+    game = GameState()
+    game.board = [["O", "X","X"],
+         ["X", "O", "O"],
+         ["X", "O", "X"]]
+    game.possibleInputs = []
+
+    with pytest.raises(SystemExit):
+        takeTurns(game, True)
```

Comment:
I like that you found a way to test the standardout text. 
But I'm not so sure about passing a boolean argument to `takeTurns` that says whether the game has been won when you already have a function that does that (`checkForWin`). What about passing in the `checkForWin` function itself or a `WinChecker` object? Then you could mock that dependency in your tests.
One nice thing about this is if you ever wanted to change to use a different `WinChecker` object, you could do it without changing `takeTurns`.

Reply:
Feature has been implemented